_getting-started/сontracts-and-workspaces.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Contracts and Workspaces
+layout: article
+section: Basic Concepts
+order: 1
+since: 20190328
+---
+
+This document provides information on [contracts](#contracts), [contract
+management](#contract-management), [workspaces](#workspaces) and [workspace
+management](#section). Additionally, it explains the basic application of this
+approach in [solution life cycle](#workspaces-and-life-cycles). The following
+scheme shows here contracts and workspaces stand in solution hierarchy.  
+
+
+
+![](media/73655d79cfa16743bcf8248345ea618b.png)
+
+## Contracts
+
+
+A client’s enclosed environment within a
+[tenant](https://docs.elastic.io/getting-started/tenant.html) is called a
```

Comment:
Please use relative links only as our white-label customers have their or domains for the docs.

Reply:
I think just `[tenant](tenant)` should work